cinema/serializers.py

```diff
@@ -68,13 +93,58 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionSerializer(read_only=True)
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+
+    def validate(self, data):
+        data = super(TicketSerializer).validate(data)
```

Comment:
The `validate` method in `TicketSerializer` should call `super().validate(data)` instead of `super(TicketSerializer).validate(data)`. The current implementation is incorrect and will raise an error.